### ISSUE TITLE
dastset of denoising task

### ISSUE THREAD

**Issue by Estherisok666** [ckkelvinchan/BasicVSR_PlusPlus#22]
In the denoising task, what are the training data set, verification data set and test data set respectively, can you provide them? Is the Set8 dataset a validation set in the code? But in Table5 and Table6 of the paper, we see that Set8 is used for testing
